field/TinyLang/Field/Test.hs

```diff
@@ -0,0 +1,54 @@
+{- A simple printer/parser test: generate a random expression and see if
```

Comment:
I did not silently delete your code! :)
It now lives in the `test/Field/Textual.hs` file and we run tests automatically (including test for the `boolean` language) on `cabal test` or `stack test`, which is convenient.
So I think we do not need this file?
Is this probably how the old commits reappeared?

Reply:
Err, `GitHub` has tricked me and made it look like it lost my comment (I hit `ctrl-W` ("cut" in emacs) in my browser again).